src/runtime/node/crypto/internal/constants.ts

```diff
@@ -0,0 +1,221 @@
+/* eslint-disable unicorn/no-useless-spread --
+ * disable eslint rules that make this file hard to recreate/update in the future */
+
+const constants = {
+  // collected from Node.js v18.20.3 using:
+  // require('crypto').constants
+  // require('crypto').constants.defaultCipherList
+  ...{
+    OPENSSL_VERSION_NUMBER: 805_306_576,
```

Comment:
(for versions same as other places i think we can explicitly set to 0 to avoid any assumptions)

```suggestion
    OPENSSL_VERSION_NUMBER: 0,
```

Reply:
so set all numbers to `0` and strings to `''`?

Reply:
I would set them for **versions** 👍🏼 (OP flags, and numbers i guess are safe unless you belive they also might make implicit behavior in polyfills)